endpoints/getting-started/openapi-appengine.yaml

```diff
@@ -0,0 +1,108 @@
+# [START swagger]
+swagger: "2.0"
+info:
+  description: "A simple Google Cloud Endpoints API example."
+  title: "Endpoints Example"
+  version: "1.0.0"
+host: "YOUR-PROJECT-ID.appspot.com"

```

Comment:
Done. It doesn't look like there is a .NET getting started example for GCE or GKE. Do we need openapi.yaml at all?